README.md

```diff
@@ -1,4 +1,8 @@
-# drake_ros
+# Drake ROS
+
+[![drake-ros continuous integration](https://github.com/RobotLocomotion/drake-ros/actions/workflows/main.yml/badge.svg?branch=develop)](https://github.com/RobotLocomotion/drake-ros/actions/workflows/main.yml)
```

Comment:
FYI it doesn't look like we can get a different badge shield per branch.

Reply:
Huh? I see two diff. badges for these two URLs:
```
https://github.com/RobotLocomotion/drake-ros/actions/workflows/main.yml/badge.svg?branch=develop
https://github.com/RobotLocomotion/drake-ros/actions/workflows/main.yml/badge.svg?branch=hidmic/ci-tweaks
```

Reply:
I also see a useful badge for your separate PR:
```
https://github.com/RobotLocomotion/drake-ros/actions/workflows/bazelized.yml/badge.svg?branch=hidmic/drake_ros_bazel_installed
```

Reply:
Right, but the badge itself is the same. The text is taken from the workflow name, regardless of which branch it was targeting.

Reply:
Oh, gotcha. Yeah, I think that's fine.
Ideally, whatever branch is shown, we just shown that status.

And yeah, if we want to show develop, I think it's fine to just make a table; perhaps use the `shields.io` variant for something a bit more compact.

Reply:
> perhaps use the shields.io variant for something a bit more compact.

[![GitHub Workflow Status (branch)](https://img.shields.io/github/workflow/status/RobotLocomotion/drake-ros/drake-ros%20continuous%20integration/develop?label=drake-ros%20CI%20%28develop%29)](https://github.com/RobotLocomotion/drake-ros/actions/workflows/main.yml)

[![GitHub Workflow Status (branch)](https://img.shields.io/github/workflow/status/RobotLocomotion/drake-ros/drake-ros%20continuous%20integration/main?label=drake-ros%20CI%20%28main%29)](https://github.com/RobotLocomotion/drake-ros/actions/workflows/main.yml)

Neat! Shall we switch to it?

Reply:
Sounds good to me!

Reply:
Ah, I just saw this. I'll do so in a follow-up PR.